core/storage/statistics/gae_models.py

```diff
@@ -98,8 +97,6 @@ class AnswerSubmittedEventLogEntryModel(base_models.BaseModel):
     client_time_spent_in_secs = ndb.FloatProperty()
     # Whether the submitted answer received useful feedback
     is_feedback_useful = ndb.BooleanProperty(indexed=True)
-    # Whether the submitted answer is the correct answer.
-    is_answer_correct = ndb.BooleanProperty(indexed=True)
```

Comment:
Btw -- how were we determining this before? I can't quite see where it's been removed at the controller level, but it's late and maybe I'm missing something...

Reply:
@seanlip As I said above, this event didn't exist before.